frontend/nuxt.config.ts

```diff
@@ -127,20 +68,18 @@ export default defineNuxtConfig({
     apiClientSecret: "",
     public: {
       // These values can be overridden by the NUXT_PUBLIC_* env variables
-      deploymentEnv,
-      apiUrl,
+      deploymentEnv: "local",
+      apiUrl: "https://api.openverse.org/",
       providerUpdateFrequency: 3600000,
       savedSearchCount: 4,
       sentry: {
         dsn: "",
-        environment: deploymentEnv,
+        environment: "local",
         release: import.meta.env.SEMANTIC_VERSION,
```

Comment:
We have to Sentry plugins: one for the nitro server, and one for Nuxt itself (client). In the second one, I don't think we can get the value of an env variable. So, I got the release from `runtimeConfig` there, too.

Reply:
I'll test, but I believe the client plugin would also be able to reference import.meta.env.SEMANTIC_VERSION and it would be statically replaced + added to the bundle at build time.